xds/src/main/java/io/grpc/xds/XdsCredentialsRegistry.java

```diff
@@ -39,10 +39,9 @@
  * Registry of {@link XdsCredentialsProvider}s. The {@link #getDefaultRegistry default
  * instance} loads providers at runtime through the Java service provider mechanism.
  */
-// TODO(anramach): Create a tracker issue for marking this experimental. 
-@ExperimentalApi("")
+@Internal
 @ThreadSafe
-public final class XdsCredentialsRegistry {
+final class XdsCredentialsRegistry {
```

Comment:
If it isn't public, then it doesn't need the `@Internal` annotation because it isn't exposed at all.

Reply:
Thanks, removed.